packages/experimental/Expander/src/Expander.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { expanderName, ExpanderType, ExpanderProps, ExpanderViewProps, ExpanderChangeEvent } from './Expander.types';
+import { compose, mergeProps, withSlots, UseSlots, buildProps } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const ExpanderComponent = ensureNativeComponent('MSFExpanderView');
+function delay(ms: number) {
+  return new Promise( resolve => setTimeout(resolve, ms) );
+}
+
+export const Expander = compose<ExpanderType>({
+  displayName: expanderName,
+  tokens: [{}, expanderName],
+  slotProps: {
+    root: buildProps(
+      (tokens) => ({...tokens})
+    ),
+  },
+  slots: { root: ExpanderComponent },
+  render: (userProps: ExpanderProps, useSlots: UseSlots<ExpanderType>) => {
+    const Root = useSlots(userProps).root;
+    const [expandedState, setExpandedState] = React.useState(userProps.expanded);
+    const expanderHeight = expandedState? userProps.expandedHeight : userProps.collapsedHeight;
+    async function onExpanderChange(event: ExpanderChangeEvent) {
+      if (event.nativeEvent.expanded != null) {
```

Comment:
In my own native component I am implementing (https://github.com/microsoft/fluentui-react-native/pull/827/files#diff-a20f47efb37c1fffa3d336f27135a753a289722fdd477b6e5c70291be7f28784R36), I ended up just doing `(event: Any)`, which according to @NickGerleman is how typescript sees this flow type anyway. You might want to consider just doing that since the type isn't actually getting read properly.